addons/scratchr2/forums.css

```
@@ -297,7 +297,7 @@ body > #pagewrapper {
 }
 .postmsg ::selection {
   /* Don't break [color=transparent] hidden text */
-  background-color: var(--darkWww-button-variant, #4280d7);
+  background-color: var(--darkWww-button-variant, #7854c0);
```

```suggestion
  background-color: #3373cc;
```
Scratch's selection color is still blue.